cfme/infrastructure/host.py

```diff
@@ -56,7 +56,7 @@ class Host(BaseEntity, Updateable, Pretty, PolicyProfileAssignable, WidgetasticT
     pretty_attrs = ['name', 'hostname', 'ip_address', 'custom_ident']
 
     name = attr.ib()
-    provider = attr.ib()
+    provider = attr.ib(default=None)
```

Comment:
A host can have no provider?

Reply:
I asked psav  the same yestarday :). Yes it can have no provider.